### PR TITLE
some minor ui changes 

### PR DESCRIPTION
**Pull request by Panth1823**

### CODE REVIEW

**Comment by Panth1823** [comment]
issue #54

File: assets/main.css [hunk 1]
```diff
@@ -160,7 +159,17 @@ main {
...
+  background-image: linear-gradient(to right, var(--purple), var(--pink));
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  transform: scale(1.3);
+}
 
 .task .actions .delete {
+  color: grey;
+}
+.task .actions .delete :hover {
```
Comment (malivinayak) [inline assets/main.css:172]: add blank line between two tags/class/id for css

File: assets/main.css [hunk 2]
```diff
@@ -160,7 +159,17 @@ main {
   -webkit-background-clip: text;
   -webkit-text-fill-color: transparent;
 }
+.task .actions .edit :hover {
```
Comment (malivinayak) [inline assets/main.css:162]: add blank line between two tags/class/id for css

**Review by malivinayak — CHANGES_REQUESTED** [review]

**Comment by Panth1823** [comment]
i uploded it by mistake @malivinayak

**Pull request closed without merging**